nyc_taxi/nyc_taxi.ipynb

```diff
@@ -440,7 +459,7 @@
    "outputs": [],
    "source": [
     "pickups = hv.Points(df, ['pickup_x', 'pickup_y'])\n",
-    "shade =  hd.datashade(pickups, aggregator=ds.count_cat('pickup_hour'), color_key=colors)\n",
+    "shade =  hd.datashade(pickups, aggregator=ds.by('pickup_hour'), color_key=colors)\n",
```

Comment:
Out of curiosity, why updating from `ds.count_cat` to `ds.by`? I'm not super up to date on Datashader's API 🙃

Reply:
It was recommended by Jim (March 20 meeting) as the more updated option in Datashader.